doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,122 @@
+
+Open the command prompt in Windows or any Linux terminal to run a curl command (as shown below) to generate the required `global_conf.json` file in your current working directory.
+
+Make sure you replace `thethings.example.com` with your server address:
+
+```bash
+$ curl -XGET \
+    "https://thethings.example.com/api/v3/gcs/gateways/{GATEWAY_ID}/semtechudp/global_conf.json" \
+    -H "Authorization: Bearer {GTW_API_KEY}" > ~/global_conf.json
+```
+
+> Note: Replace the required fields in the above command and run it.
+
+Once the `global_conf.json` file is generated, you will need to add this to your gateway. In a web browser, open the gateway’s configuration page by navigating to its IP Address obtained from the network it is connected to. Once logged in, you can configure the gateway to connect to {{%tts%}} by following the steps below:
+
+- Click on **LoRaWAN<sup>®</sup>** in the menu on the left. It opens the Gateway&apos;s configuration page.
+
+{{< figure src="005_Gateway_Menu_LoRaWAN.png" alt="MultiTech Conduit gateway home page" >}}
+
+- Under **Network Settings**, select the mode as &quot;Packet Forwarder&quot;.
+
+{{< figure src="006_Packet_Forwarder_Gateway.png" alt="MultiTech Conduit packet forwarder settings" >}}
+
+- On the right side of the &quot;LoRa Packet Forwarder Configuration&quot; section, you can find &quot;Manual Configuration &quot;. Click on it to setup the channel plan manually.
+
+{{< figure src="001_Gateway_Frequency_plan_manual_config.png" alt="Switch to manual configuration mode" >}}
+
+- The above step will lead you to the gateway configuration editor tagged as &quot;Config&quot;.
+
+{{< figure src="002_Gateway_Frequency_plan_manual_config.png" alt="edit the configuration" >}}
+
+- Copy the contents of the `global_conf.json` file downloaded earlier and paste them in the gateway console configuration editor.
+
+- Once pasted, modify the value of `clksrc` to set it to `0`.
+
+- Click on **Submit** to save the configuration.
+
+- Now, click on **Save and Restart** from the menu.
+
+{{< figure src="011_Gateway_Menu_Save-And-Restart.png" alt="Saving the Network Interfaces Configuration" >}}
+
+- You will be prompted to confirm the restart. Choose **OK** to proceed.
+
+{{< figure src="012_System_Confirmation_Prompt.png" alt="Configuration restart prompt" >}}
+
+This will apply the custom settings and reboot the gateway. If all the steps have been followed correctly, your gateway will now connect to {{%tts%}}.
+
+> To know more about other features of the MultiTech Conduit gateway, you can refer to the **mPower Edge AEP software guide** on the [Multitech Website](http://www.multitech.net/developer/products/multiconnect-conduit-platform/conduit/).
+
+## Troubleshooting
+
+### Packet Forwarder Logs
+
+Packet forwarder logs contain the messages received from the devices and the network server. You can use the information to debug the issues.
```

Comment:
Not really. The logs contain only information _on_ the messages it received, not the actual message.